client/src/context/useSocketContext.tsx

```diff
@@ -27,5 +27,9 @@ export const SocketProvider: FunctionComponent = ({ children }): JSX.Element =>
 };
 
 export function useSocket(): ISocketContext {
+  const context = useContext(SocketContext);
+  if (context === undefined) {
+    throw new Error('useSocket must be used within a SocketProvider');
+  }
   return useContext(SocketContext);
 }
```

Comment:
You can just return the `context` variable here.

Reply:
Yeah, I forgot to do that. Thanks